packages/xo-web/src/xo-app/vm/index.js

```diff
@@ -175,14 +176,17 @@ export default class Vm extends BaseComponent {
       <Container>
         <Row>
           <Col mediumSize={6} className='header-title'>
-            <h2>
-              {isEmpty(vm.current_operations) ? (
-                <Icon icon={`vm-${vm.power_state.toLowerCase()}`} />
-              ) : (
-                <Icon icon='vm-busy' />
-              )}{' '}
-              <Text value={vm.name_label} onChange={this._setNameLabel} />
-            </h2>
+            <Copiable tagName='div'>
+              <h2 style={{ display: 'inline' }}>
+                {isEmpty(vm.current_operations) ? (
+                  <Icon icon={`vm-${vm.power_state.toLowerCase()}`} />
+                ) : (
+                  <Icon icon='vm-busy' />
+                )}{' '}
+                <Text value={vm.name_label} onChange={this._setNameLabel} />
+              </h2>{' '}
+              (<strong>{_('uuid')}</strong>: {vm.uuid})
```

Comment:
I'd replace this whole line with `<span className='text-muted'>{vm.uuid}</span>`. What do you think?

Reply:
Without the UUID label, the user will not know what means the String.